ex2/ex2.js

```diff
@@ -1,32 +1,43 @@
 /*
  * Write a function that accepts a string and a number (n) and returns the index of the n-th comma in that string.
- * If there's no such comma, the function should return -1. If the n argument is invalid, the function should throw an Error.
+ * If there's no such comma, the function should return -1. If the n argument is incommaIndexid, the function should throw an Error.
```

Comment:
Please look at the changes on github before sending for review (this change was because of a search-replace)